### ISSUE TITLE
UI improvements to the presentation of audio/video/image annotations in the roll-over

### ISSUE THREAD

**Issue by blalbrit** [ProjectMirador/mirador#1263]
Stub ticket for now

**Comment by csbailey5t** [comment]
Right now, the position of the annotation viewer on rollover is determined in relation to the mouse: https://github.com/ProjectMirador/mirador/blob/2.1.4/js/src/annotations/annotationTooltip.js#L132-L135

Similar to the question of where the annotation could be moved to such that it doesn't overlay the annotation body (in https://github.com/ProjectMirador/mirador/issues/1260), what would be a reasonable place to initialize the viewer on roll-over?

We might be able to approach this question more clearly by elaborating what the use-case is such that the user explicitly wants the annotation viewer open while being able to see the entire annotation body/ies. Any further description of such a use-case @blalbrit ?

**Comment by versae** [comment]
In addition, for multiple regions annotations is not that clear where the annotation viewer on rollover should go.

**Comment by blalbrit** [comment]
@csbailey5t and @versae . There are several issues that need to be unpacked here: @aeschylus and @snydman can elaborate on some of them.

- my understanding is that the video view box on roll-over actually cuts off a portion of the video (@snydman, @aeschylus )
- For positioning, if we move to a side-panel view (on roll-over, for instance), then I think that would work. I realize it is hard to randomly position the annotation body based on a number of factors.
- For positioning, as well, I think it would be good to understand from the Mirador community ( @aeschylus - can you summarize?) 

As far as a use-case goes, one is:
* As a curator, I want to draw a target on a specific decoration on the page and link that to a video that can be played describing that decoration in more detail. For that reason, I don't want the video to overlay the decoration, but be in a consistent place (even off the page) so that the viewer can refer to the target of the annotation while viewing the supplemental information in the video.

@versae - I agree that for multiple targets positioning is tricky for both video and non-video annotation bodies. A side-panel solution solves both instances.

**Comment by camillevilla** [comment]
This ticket is being closed as part of the Mirador 3 issues review. We encourage all Mirador 2 implementers to try out Mirador 3 and leave feedback in new tickets.
